include/service/entities/user_service.js

```diff
@@ -136,31 +136,50 @@ module.exports = function UserServiceModule(pb) {
 
         return adminOptions;
     };
-
+    
     /**
      * Retrieves a select list (id/name) of available system editors
+     * @deprecated since 0.4.0
      * @method getEditorSelectList
      * @param {String} currId The Id to be excluded from the list.
      * @param {Function} cb A callback that takes two parameters.  The first is an
      * error, if exists, the second is an array of objects that represent the
      * editor select list.
      */
-    UserService.prototype.getEditorSelectList = function(currId, cb) {
+    UserService.prototype.getEditorSelectList(currId, cb) {
+        pb.log.warn('UserService: getEditorSelectList is deprecated. Use getWriterOrEditorSelectList instead');
+        this.getWriterOrEditorSelectList(currId, cb);
+    };
```

Comment:
I added the function signature back.  I did this because we don't know if someone was already using it.  I'd like to a take a deprecation strategy where possible to reduce churn on our users.